pkg/extension/sumologicextension/extension.go

```diff
@@ -44,6 +45,7 @@ const (
 	heartbeatUrl             = "/api/v1/collector/heartbeat"
 	registerUrl              = "/api/v1/collector/register"
 	DefaultHeartbeatInterval = 15 * time.Second
+	collectorCredentialsPath = "/tmp/sumo_otc_creds"
```

Comment:
Mixed block of exported and not exported variables doesn't look good to me. I suggest either splitting into two blocks or changing order  of them.